include/og/Screen/ogScreen.h

```diff
@@ -109,6 +110,10 @@ struct DispMemberBase {
 	DispMemberBase* m_subMember; // _04
 };
 
+extern const char* PikiIconTextureName[19];
```

Comment:
no need to extern in headers, as that is implicit in the declaration iirc

Reply:
I was under the impression that externing in headers was just useful for generating headers for other files (for decomp.me, etc), and not putting extern would gen the instance in the first file linked using the header